jvc_network.py

```diff
@@ -23,11 +23,12 @@ class Timeout(Exception):
 
 class JVCNetwork:
     """JVC projector network connection"""
-    def __init__(self, print_all=False, print_recv=False, print_send=False):
+    def __init__(self, host_port_str=None, print_all=False, print_recv=False, print_send=False):
```

Comment:
Why is this passed as an encoded string? Why not pass the config dict or host_port tuple?